pkg/render/manager.go

```diff
@@ -105,12 +107,6 @@ func Manager(
 	tlsAnnotations[tlsSecretHashAnnotation] = AnnotationHash(tlsKeyPair.Data)
 
 	if management {
-		// If there is no secret create one and add it to the operator namespace.
-		if tunnelSecret == nil {
-			tunnelSecret = voltronTunnelSecret()
-			tlsSecrets = append(tlsSecrets, tunnelSecret)
```

Comment:
Is this line not needed too?

Reply:
This copied the secret from the operator namespace. Since we are not generating it, we should only render the secret that is copied inside the tigera-manager namespace.